pkg/api/resource/resource_helpers.go

```diff
@@ -143,10 +113,10 @@ func GetNodeCondition(status *NodeStatus, conditionType NodeConditionType) (int,
 	return -1, nil
 }
 
-// Updates existing pod condition or creates a new one. Sets LastTransitionTime to now if the
+// UpdatePodCondition updates existing pod condition or creates a new one. Sets LastTransitionTime to now if the
```

Comment:
Hmm, i'm just blindly moving the entire file, these functions are in the resource_helpers.go already: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/resource_helpers.go#L149.